### ISSUE TITLE
Effect Value field replaces the user's entry with "0"

### ISSUE THREAD

**Issue by beck99an** [kandashi/Dynamic-Effects-SRD#8]
![image](https://user-images.githubusercontent.com/5571794/97328118-1131f580-184c-11eb-894b-8ce97549f9a2.png)

When editing an active effect opened from an item in the items library, values entered in the "Effect Value" field change to 0 when tabbing or clicking outside the field.

I have reproduced this behaviour on another item pulled from the DAE SRD compendium as well as a user created item.

(@zakalwe#3167 on discord)

**Comment by kandashi** [comment]
I cant replicate this in my testing environment, could you disable modues apart from DAE and the SRD and confirm it still occurs

**Comment by beck99an** [comment]
I can't replicate it with only DAE and the SRD enabled - if it's helpful, let me know and I'll run through my modules one-by-one to see if the issue is replicated.

![image](https://user-images.githubusercontent.com/5571794/97374886-c8039500-188f-11eb-821d-b0cb1f04dbb1.png)
![image](https://user-images.githubusercontent.com/5571794/97374899-d18cfd00-188f-11eb-868e-59cd939fead3.png)

**Comment by kandashi** [comment]
I can rule out a lot of those as I use them myself, I would be interested about Math.js, Skys alt sheet and Monster blocks. But if its not one of those it would be usefull to find out which one it is. I imagine this is an issue with DAE itself and not the SRD.

**Comment by beck99an** [comment]
Well, go figure. I re-enabled those modules and can't replicate. Enabled all modules and still can't replicate. Edge case I guess - can't figure out what I'm doing differently now. I restarted the machine hosting the game before trying to replicate. Not sure if that might have fixed something - it had been up for some time. In fact, I was having issues with the dropdown menus not persisting (I had to hold left click to keep the menu open when first selecting something) and that issue is also no longer present. If I can help further, let me know.
